roomies-server/app.js

```diff
@@ -62,39 +62,39 @@ app.post("/joinHouse", authenticateJWT, (req, res) => {
  * Ticket: https://trello.com/c/JNVH6ycc
  * This API return all items in the grocery list
  */
-app.get("/getAllItems", (req, res) => {
+app.get("/getAllItems", authenticateJWT, (req, res) => {
```

Comment:
This route should only be accessed after the user has signed in + join a house. You could add this middleware `verifyUserInHouse` in the `house.js` .  

The method should look something like this 
```js
app.get("/getAllItems", authenticateJWT, verifyUserInHouse, (req, res) => {...}
```